notebooks/EOPF_Tutorials/Sentinel-1_L2_OCN_product_format_prototype.ipynb

```diff
@@ -0,0 +1,570 @@
+{
```

Comment:
<p>Missing ESA banner at the end of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EOPF-Sample-Service/eopf-sample-notebooks/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>